src/EnergyPlus/Plant/Component.cc

```diff
@@ -114,16 +125,7 @@ namespace DataPlant {
         // If you add a module or new equipment type, you must set up this structure.
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        if (InitLoopEquip) {
-
-            this->compPtr->onInitLoopEquip(state, this->location);
-            this->compPtr->getDesignCapacities(state, this->location, this->MaxLoad, this->MinLoad, this->OptLoad);
-            this->compPtr->getDesignTemperatures(this->TempDesCondIn, this->TempDesEvapOut);
 
-            if (GetCompSizFac) {
-                this->compPtr->getSizingFactor(this->SizFac);
-            }
-        }
         this->compPtr->simulate(state, this->location, FirstHVACIteration, this->MyLoad, this->ON);
```

Comment:
Next step is definitely just calling the compPtr->simulate() function from wherever the CompData::simulate method was called to continue streamlining this.

This is the type of positivity we all need :100: